ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -414,7 +595,11 @@ def _get_free_ports(client):
                             exclude_ports.append(node.messaging_port)
                 except:
                     logger.error('  Could not load port information of cluster {0}'.format(cluster_name))
-        ports = System.get_free_ports(EtcdConfiguration.get('/ovs/framework/hosts/{0}/ports|arakoon'.format(node_name)), exclude_ports, 2, client)
+        key = '/ovs/framework/hosts/{0}/ports|arakoon'.format(node_name)
```

Comment:
This change might become obsolete if you call the etcdconfiguration's initialize before you start using the arakooninstaller. Then the ```ARAKOON_START_PORT``` isn't needed either. And you have the benefit that you might be able to adapt the initialize method to overwrite arakoon ports before, so you can setup clusters on other ports.